### DISCUSSION TITLE
Full List of possible emojis and their short names?

### DISCUSSION THREAD

**Discussion by taijidude** [Textualize/rich#1323] [Q&A] [answered]
Hello, 

i see only some emojis in the documentation example. Ist there a full list, of which emojis are possible? Maybe with their short names?

**Comment by willmcgugan** [comment, marked as answer]
```
python -m rich.emoji
```

**Reply by taijidude** [reply]
Thank you! :-)

**Reply by heyajulia** [reply]
With uvx:

```bash
uvx --from rich -- python3 -m rich.emoji
```

**Comment by NikosAlexandris** [comment]
Would expanding the default list of emojis be considered? I.e. not having to attempt to add new emojis manually.

**Reply by willmcgugan** [reply]
I’d just a big list. I’d accept a PR with an update.
